app/routes/labs/route.tsx

```diff
@@ -26,7 +26,7 @@ export default function () {
     <>
       <Header />
       <h1>ACROSS</h1>
-      <main id="main-content">
+      <main id="main-content" className="bg-ink text-white">
```

Comment:
Would you please move this to theme.scss as a class called `usa-prose--dark` so that we can keep the dark mode theming separate from the content?